commands/export.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
 		let tups = (await bot.db.query('SELECT name, avatar_url, brackets, posts, show_brackets, birthday, description, tag, group_id, group_pos FROM Members WHERE user_id = $1 ORDER BY position', [msg.author.id])).rows;
 		let groups = (await bot.db.query('SELECT id, name, description, tag FROM Groups WHERE user_id = $1 ORDER BY position',[msg.author.id])).rows;
 		let data = { tuppers: tups, groups };
-		return bot.send(msg.channel,"",{name:"tuppers.json",file:Buffer.from(JSON.stringify(data))});
+		let channel = await msg.author.getDMChannel(); //get the user's DM channel
+		return channel.send(msg.channel,"",{name:"tuppers.json",file:Buffer.from(JSON.stringify(data))}); //send it to them in DMs
```

Comment:
need a try/catch around these two lines to handle users having DMs blocked, if DMs are blocked, then send it in the channel instead